ontology/yaml/resources/PLUMBING/entity_types/VLV.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+########################
+### Canonical Types  ###
+########################
+
+VLV_SWCM:
```

Comment:
These are indistinguishable from a data perspective. I don't think we need two separate types for these; reuse VLV_MWCM.

Reply:
I see sir. I'll use WVCM (water valve control / monitor) and so it'll be: VLV_WVCM